docs/.vuepress/theme/components/Footer.vue

```diff
@@ -17,6 +18,29 @@
   </footer>
 </template>
 
+<script>
+  export default {
+    computed: {
+      lastUpdatedDate() {
+        const moment = require('moment')
+
+        function compareTimestamps(a, b) {
```

Comment:
Let's avoid declaring this function every time in the computed property. Below this move this to the `methods` object of your component:
```
    methods: {
      compareTimestamps: function(a, b) {
        if (a.lastUpdated < b.lastUpdated)
          return -1;
        if (a.lastUpdated > b.lastUpdated)
          return 1;
        return 0;
      }
    },
```
Then from this computed property, you can call `this.compareTimestamps`

Reply:
Oh yeah thanks, I'm pretty new to Vue.js and didn't think about this.
The changes you've asked for are in this commit: https://github.com/ethereum/ethereum-org-website/pull/356/commits/483f17d81c53bc800a9f039c7f5e89daf1c056f7